packages/dcos-net/extra/dcos-net-observer.py

```diff
@@ -0,0 +1,26 @@
+#!/opt/mesosphere/bin/python
+
+import http.client
+import sys
+import time
+
```

Comment:
Can we add some comments here to explain the purpose of this file.

Reply:
https://github.com/mesosphere/dcos/blob/urbanserj/dcos-net/packages/dcos-net/extra/dcos-net-observer.py#L3-L9